crypto/kzg4844/trusted_setup.json

```diff
@@ -4164,4 +4164,4 @@
     "0xa92039a08b5502d5b211a7744099c9f93fa8c90cedcb1d05e92f01886219dd464eb5fb0337496ad96ed09c987da4e5f019035c5b01cc09b2a18b8a8dd419bc5895388a07e26958f6bd26751929c25f89b8eb4a299d822e2d26fec9ef350e0d3c",
     "0x92dcc5a1c8c3e1b28b1524e3dd6dbecd63017c9201da9dbe077f1b82adc08c50169f56fc7b5a3b28ec6b89254de3e2fd12838a761053437883c3e01ba616670cea843754548ef84bcc397de2369adcca2ab54cd73c55dc68d87aec3fc2fe4f10"
   ]
-}
\ No newline at end of file
+}
```

Comment:
no need to change this files?

Reply:
reverted in [4b14913](https://github.com/scroll-tech/go-ethereum/pull/932/commits/4b14913353b35face8b1ac0cbfcfce6a2960614e).